### PR TITLE
template(kingworld): add kingworld template

### PR DESCRIPTION
**Pull request by SaltyAom**
Add template for [kingworld](https://github.com/saltyaom/kingworld) to use with `bun create`

### CODE REVIEW

**Comment by Kapsonfire** [comment]
yesterday they wanted me to copy the gitignore from next example
and dont name it .gitignore but `gitignore` without dot

File: examples/kingworld/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "name": "kingworld",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon --config nodemon.json"
+  },
+  "keywords": [],
+  "author": "",
```
Comment (FinnRG) [inline examples/kingworld/package.json]: Please remove these empty fields

File: examples/kingworld/gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+node_modules
```
Comment (FinnRG) [inline examples/kingworld/gitignore]: Could you use the same gitignore as the ``next`` template? It's just to have some consistency between the templates.

File: examples/kingworld/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# KingWorld Starter
+
+## Scaffold
+To get start with this template, simply paste the command in to your terminal:
+```bash
+bun create kingworld
```
Comment (FinnRG) [inline examples/kingworld/README.md]: Please change this to ``bun create kingworld ./kingworld-example`` to emphasize that the user can choose their own folder.

File: examples/kingworld/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+```
+
+## Development
+KingWorld is set with `nodemon` for auto reload, simply run:
+```bash
+bun run dev
+```
+
+Go to http://localhost:3000 and enjoy your new awesome web server
```
Comment (FinnRG) [inline examples/kingworld/README.md]: Please use ``Open http://localhost:3000/ with your browser to see the result.`` like the other templates.

File: examples/kingworld/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# KingWorld Starter
```
Comment (FinnRG) [inline examples/kingworld/README.md]: Please use ``KingWorld with Bun runtime`` like the other templates.

File: examples/kingworld/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+## Scaffold
```
Comment (FinnRG) [inline examples/kingworld/README.md]: Please use ``Getting Started``  instead of  ``Scaffold`` like the other templates.

**Review by FinnRG — CHANGES_REQUESTED** [review]
Thank you for your work. Your framework looks quite promising (I also like how you chose the name :D). I have only some minor consistency issues.

File: examples/kingworld/package.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  "license": "ISC",
+  "dependencies": {
+    "kingworld": "latest"
+  },
+  "devDependencies": {
+    "bun-types": "^0.1.4",
+    "nodemon": "^2.0.19"
+  }
+}
```
Comment (Mr0nline) [inline examples/kingworld/package.json]: There should be EOFL!

File: examples/kingworld/README.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# KingWorld with Bun runtime
+
+## Getting Started
+To get start with this template, simply paste the command in to your terminal:
```
Comment (FinnRG) [inline examples/kingworld/README.md]: To get started with this template, simply paste this command into your terminal:

File: examples/kingworld/README.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+```bash
+bun create kingworld ./kingworld-example
+```
+
+## Development
+KingWorld is set with `nodemon` for auto reload, simply run:
```
Comment (FinnRG) [inline examples/kingworld/README.md]: KingWorld uses `nodemon` for auto reload. To start the development server run:

**Review by FinnRG — CHANGES_REQUESTED** [review]

**Review by FinnRG — APPROVED** [review]

**Comment by FinnRG** [comment]
LGTM, thank you for your work.

**Pull request merged**